source/geod24/concurrency.d

```diff
@@ -480,7 +518,7 @@ if (isSpawnable!(F, T))
     }
 
     auto tid1 = spawn(&f1, "Hello World");
-    thread_joinAll;
+    Thread.sleep(1000.msecs);
```

Comment:
Does `thread_joinAll` not work anymore?

Reply:
When `thread_joinAll` is used, it blocks without shutting down.